includes/admin/metaboxes/settings.php

```diff
@@ -522,6 +522,25 @@ private static function calendar_settings_panel($post)
 				</tr>
 			</thead>
 			<tbody class="simcal-panel-section">
+				<tr class="simcal-panel-field">
+					<th>
+						<label for="_display_print_calendar"><?php _e('Print Calendar', 'google-calendar-events'); ?></label>
+					</th>
+					<td>
+						<?php
+      $fixed = get_post_meta($post->ID, '_display_print_calendar', true);
```

Comment:
$fixed can be replaced with a more verbose constant name like $display_print_calendar_value